apps/condo/domains/miniapp/components/GlobalApps/GlobalAppsContainer.tsx

```diff
@@ -126,8 +130,8 @@ export const GlobalAppsContainer: React.FC = () => {
         }
     }, [user, loading])
 
-    // // Global miniapps allowed only for authenticated users
-    if (!user) {
+    // // Global miniapps allowed only for authenticated (not support or admin) users
+    if (!user || user.isSupport || user.isAdmin) {
```

Comment:
Why admin or support can't use global miniapps?

Reply:
Because you can steal an admin token or support token from a third-party application.
For not global miniapps we have such check (B2BAppPage).